proto/storage/image_component.proto

```diff
@@ -7,6 +7,9 @@ import "storage/image.proto";
 option go_package = "./storage;storage";
 option java_package = "io.stackrox.proto.storage";
 
+// ******************************
+// This proto is deprecated.
+// ******************************
 message ImageComponent {
```

Comment:
Or change the comment so it will be properly rendered in godoc marking this message as deprecated and pointing to V2
```suggestion
message ImageComponent {
   option deprecated = true;
```

Reply:
TIL